### PR TITLE
zh: update get-started.md for 2.0

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed, added or deleted? (Required)
update install version and config for 2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
preview: https://github.com/GMHDBJD/docs-dm/blob/get-started-2.0/zh/get-started.md

### CODE REVIEW

File: zh/get-started.md [hunk 1]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
+       244 pts/0    00:00:03 dm-master
+       245 pts/0    00:00:02 dm-master
+       283 pts/0    00:00:00 dm-worker
+       284 pts/0    00:00:00 dm-worker
+       285 pts/0    00:00:00 dm-worker
+       324 pts/0    00:00:00 ps
     ```
 
-每个上游的 MySQL Server 实例对应一个单独的 DM-worker 实例，每个 DM-worker 实例都有各自的配置文件。这些文件内容包括：
+3. 使用 dmctl 将数据源配置加载到 DM 集群中，执行如下命令并得到相应的返回:
```
Comment (toutdesuite) [inline zh/get-started.md:274]:
```suggestion
3. 使用 dmctl 将数据源配置加载到 DM 集群中，执行如下命令并得到相应的返回：
```

File: zh/get-started.md [hunk 2]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
+                "worker": "worker3"
+            }
+        ]
+    }
+    ```
 
-- 连接到上游 MySQL Server 的详细信息
-- relay log（上游服务器的 binlog）的存储路径
-- mydumper 的输出
+每个上游的 MySQL Server 实例对应一个单独的 source.toml。每个 DM-master ， DM-worker 实例都有各自的配置文件。
```
Comment (toutdesuite) [inline zh/get-started.md:323]:
```suggestion
每个上游的 MySQL Server 实例对应一个单独的 source.toml。每个 DM-master、DM-worker 实例都有各自的配置文件。
```

File: zh/get-started.md [hunk 3]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
 以下为 `dm-worker1.toml` 的示例：
 
 {{< copyable "" >}}
 
 ```toml
-# DM-worker 配置
+# DM-worker1 Configuration
+
+name = "worker1"
+worker-addr="0.0.0.0:8262"
```
Comment (toutdesuite) [inline zh/get-started.md:347]:
```suggestion
worker-addr = "0.0.0.0:8262"
```

File: zh/get-started.md [hunk 4]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
+advertise-addr="127.0.0.1:8262"
```
Comment (toutdesuite) [inline zh/get-started.md:348]:
```suggestion
advertise-addr = "127.0.0.1:8262"
```

File: zh/get-started.md [hunk 5]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
-
+timezone: "Asia/Shanghai"
+ 
 target-database:
   host: "127.0.0.1"
   port: 4000
   user: "root"
   password: ""
-
+
```
Comment (toutdesuite) [inline zh/get-started.md:388]:
```suggestion

```

File: zh/get-started.md [hunk 6]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
   port: 4000
   user: "root"
   password: ""
-
+ 
 mysql-instances:
-  - source-id: "mysql1"
-    black-white-list: "dmtest1"
+  - source-id: "mysql-replica-01"
+    black-white-list:  "instance"
```
Comment (toutdesuite) [inline zh/get-started.md:391]:
```suggestion
    black-white-list: "instance"
```

File: zh/get-started.md [hunk 7]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
+ 
+source-id = "mysql-replica-01"
+ 
 [from]
 host = "127.0.0.1"
 user = "root"
 password = ""
 port = 3307
-```
+`````
```
Comment (toutdesuite) [inline zh/get-started.md:366]: 这里需要删去多余的反引号

File: zh/get-started.md (hunk 5 above)
Comment (toutdesuite) [inline zh/get-started.md:388]: 这里需要删除多余的空格

File: zh/get-started.md [hunk 8]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
     loader-config-name: "loader2"
-  - source-id: "mysql3"
-    black-white-list: "dmtest1"
+ 
+  - source-id: "mysql-replica-03"
+    black-white-list:  "instance"
+    mydumper-config-name: "global"
     loader-config-name: "loader3"
-
+
```
Comment (toutdesuite) [inline zh/get-started.md:404]: 这里需要删除多余的空格

File: zh/get-started.md [hunk 9]
```diff
@@ -246,82 +258,157 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
-- 如果从 MariaDB Server 或 MariaDB (Galera) Cluster 迁移数据，则设置 `flavor = "mariadb"`（仅支持 10.1.2 以上 MariaDB 版本）。
-- 从 DM 1.0.2 版本开始，`flavor`、`server-id` 项均会由 DM 自动生成，一般情况下不需要手动配置。
 - `from` 中的 `password` 如果不为空，则需要使用 dmctl 进行加密，参见[使用 dmctl 加密上游 MySQL 用户密码](deploy-a-dm-cluster-using-ansible.md#使用-dmctl-加密上游-mysql-用户密码)。
 
 任务在 YAML 文件中定义。以下为一个 `dmtask1.yaml` 文件示例：
 
 {{< copyable "" >}}
 
 ```yaml
+---
```
Comment (toutdesuite) [inline zh/get-started.md:375]: 这里需要删去“---”

Comment (GMHDBJD) [reply zh/get-started.md:375]: 这是语法

Comment (toutdesuite) [reply zh/get-started.md:375]: 好的，了解了

File: zh/get-started.md [hunk 10]
```diff
@@ -344,7 +431,9 @@ loaders:
 
 * `black-white-list`：将一个任务限制在数据库 `dmtest` 中。
 
-* `loaders`：定义由各个 DM-worker 实例执行的每个 mydumper 实例的输出地址。
+* `mydumpers`: 定义各个 MySQL 源的 mydumper 二进制文件路径。
```
Comment (toutdesuite) [inline zh/get-started.md:434]:
```suggestion
* `mydumpers`：定义各个 MySQL 源的 mydumper 二进制文件路径。
```

File: zh/get-started.md [hunk 11]
```diff
@@ -344,7 +431,9 @@ loaders:
 
 * `black-white-list`：将一个任务限制在数据库 `dmtest` 中。
 
-* `loaders`：定义由各个 DM-worker 实例执行的每个 mydumper 实例的输出地址。
+* `mydumpers`：定义各个 MySQL 源的 mydumper 二进制文件路径。
```
Comment (WangXiangUSTC) [inline zh/get-started.md:434]: 新版本不再需要配置 mydumper 二进制文件路径了吧

File: zh/get-started.md [hunk 12]
```diff
@@ -207,16 +207,26 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
     ```bash
     tidb-server --log-file=logs/tidb-server.log &
-    for i in 1 2 3; do dm-worker --config=dm-cnf/dm-worker$i.toml & done
-    dm-master --config=dm-cnf/dm-master.toml &
+    ```
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    for i in 1 2 3; do dm-master --config=dm-cnf/dm-master"$i".toml --log-file=logs/dm-master"$i".log >> logs/dm-master"$i".log 2>&1 & done
```
[the remark this answers is not in the document]
Comment (GMHDBJD) [reply zh/get-started.md:223]: 这个已经改了，merge进去后就正确了。

File: zh/get-started.md [hunk 13]
```diff
@@ -246,83 +258,151 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
+       243 pts/0    00:00:02 dm-master
+       244 pts/0    00:00:03 dm-master
+       245 pts/0    00:00:02 dm-master
+       283 pts/0    00:00:00 dm-worker
+       284 pts/0    00:00:00 dm-worker
+       285 pts/0    00:00:00 dm-worker
+       324 pts/0    00:00:00 ps
+    ```
+
+3. 使用 dmctl 将数据源配置加载到 DM 集群中，执行如下命令并得到相应的返回：
```
Comment (yikeke) [inline zh/get-started.md:274]:
```suggestion
3. 使用 dmctl 将数据源配置加载到 DM 集群中，执行如下命令并得到相应的返回结果：
```

**Comment by GMHDBJD** [comment]
> This PR adds the `assets/get-started/dm-cnf.tgz` file. Did you do it on purpose? If not, please delete this extra file. @GMHDBJD

Yes, it needs to be updated.

File: zh/get-started.md [hunk 14]
```diff
@@ -246,83 +258,151 @@ TiDB Data Migration 平台由 3 部分组成：DM-master、DM-worker 和 dmctl
...
-- mydumper 的输出
+每个上游的 MySQL Server 实例对应一个单独的 source.toml。每个 DM-master、DM-worker 实例都有各自的配置文件。
 
-各个 DM-worker 通过不同的端口监听（由 `worker-addr` 定义）。
+以下为 `dm-master1.toml` 的示例：
+
+{{< copyable "" >}}
+
+```toml
+# DM-Master1 Configuration.
```
Comment (toutdesuite) [inline zh/get-started.md:330]:
```suggestion
# DM-master1 Configuration.
```

**Comment by toutdesuite** [comment]
@yikek PTAL. Thanks~

**Comment by yikeke** [comment]
> > This PR adds the `assets/get-started/dm-cnf.tgz` file. Did you do it on purpose? If not, please delete this extra file. @GMHDBJD
> 
> Yes, it needs to be updated.

assets/get-started/dm-cnf.tgz 这个不需要删掉吗

**Comment by GMHDBJD** [comment]
> > > This PR adds the `assets/get-started/dm-cnf.tgz` file. Did you do it on purpose? If not, please delete this extra file. @GMHDBJD
> > 
> > 
> > Yes, it needs to be updated.
> 
> assets/get-started/dm-cnf.tgz 这个不需要删掉吗 @GMHDBJD

这个之前的commit就有了，这次更新了新的配置。
https://github.com/pingcap/docs-dm/blame/ef7230fee66d9355b49ce9650dbea2ae57c6d85f/zh/get-started.md#L55

**Pull request merged**